test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'test_helper'
```

Comment:
Replace with a system test
Usually we only test models and the whole system. Integration is only needed in case of api